ui-library-click-to-call/README.md

```diff
@@ -47,3 +53,1123 @@ Instead, it will copy all the configuration files and the transitive dependencie
+
+`NewWindowCallScreen.tsx`
+
+```typescript
+export const SameOriginCallScreen = (props: {
+  adapterArgs: {
+    userId: CommunicationUserIdentifier;
+    displayName: string;
+    token: string;
+    locator: CallAdapterLocator;
+    alternateCallerId?: string;
+  };
+  useVideo: boolean;
+}): JSX.Element => {
+  const { adapterArgs, useVideo } = props;
+
+  const credential = useMemo(() => {
+    try {
+      return new AzureCommunicationTokenCredential(adapterArgs.token);
+    } catch {
+      console.error("Failed to construct token credential");
+      return undefined;
+    }
+  }, [adapterArgs.token]);
+
+  const args = useMemo(() => {
+    return {
+      userId: adapterArgs.userId,
+      displayName: adapterArgs.displayName,
+      credential,
+      token: adapterArgs.token,
+      locator: adapterArgs.locator,
+      alternateCallerId: adapterArgs.alternateCallerId,
+    };
+  }, [
+    adapterArgs.userId,
+    adapterArgs.displayName,
+    credential,
+    adapterArgs.token,
+    adapterArgs.locator,
+    adapterArgs.alternateCallerId,
+  ]);
+
+  console.log(args);
+
+  const afterCreate = (adapter: CallAdapter): Promise<CallAdapter> => {
+    adapter.on("callEnded", () => {
+      adapter.dispose();
+      window.close();
+    });
+    adapter.joinCall(true);
+    return new Promise((resolve, reject) => resolve(adapter));
+  };
+
+  const adapter = useAzureCommunicationCallAdapter(args, afterCreate);
+
+  if (!adapter) {
+    return (
+      <Stack
+        verticalAlign="center"
+        styles={{ root: { height: "100vh", width: "100vw" } }}
+      >
+        <Spinner
+          label={"Creating adapter"}
+          ariaLive="assertive"
+          labelPosition="top"
+        />
+      </Stack>
+    );
+  }
+  return (
+    <Stack styles={{ root: { height: "100vh", width: "100vw" } }}>
+      <CallComposite
+        options={{
+          callControls: {
+            cameraButton: useVideo,
+            screenShareButton: useVideo,
+            moreButton: false,
+            peopleButton: false,
+            displayType: "compact",
+          },
+          localVideoTileOptions: {
+            position: !useVideo ? "hidden" : "floating",
+          },
+        }}
+        adapter={adapter}
+      />
+    </Stack>
+  );
+};
+```
+
+For our CallComposite we have some configuration to do for Click to Call. Depending on your use case we have a number of customizations that can change the user experience. This sample chooses to hide the local video tile, camera, and screen sharing controls if the user opts out of video for their call. Other customizations seen include in the `afterCreate` function defined in the snippet we auto join the call, this will bypass the configuration screen and drop the user into the call with their mic live. Just remove the call to `adapter.join(true)` and the configuration screen will show as normal. Next we will talk about how to get this screen the information once we have our CallComposite configured.
```

Comment:
nit: For our CallComposite, we have ...
Depending on your use case, we have ...